scripts/haproxy

```diff
@@ -0,0 +1,221 @@
+#!/bin/python3
+# [This file is part of the zabbix-agent-osso package]
+# Requires: Python
+
+import argparse
+import json
+import os
+import socket
+from stat import S_ISSOCK
+
+HAPROXY_RUNDIR = '/var/run/haproxy'
+
+
+def send_command(socket_path, command):
+    """
+    Send command to socket.
+
+    Returns None on ConnectionRefusedError.
+    """
+    with socket.socket(socket.AF_UNIX, socket.SOCK_STREAM) as client:
+        try:
+            client.connect(socket_path)
+            client.sendall(command.encode())
+
+            response = client.recv(65535)  # Try to read all at once.
```

Comment:
This only works as long as the response is at most 65535 bytes. You really have to read all the data from the socket.